tests/checks/test_lifecycle_checks.py

```diff
@@ -52,7 +54,7 @@ def test_check_file_lifecycle_status(self, mock_search_metadata, mock_get_metada
 
         expected_lifecycle_statuses = {
             "file_1": DEEP_ARCHIVE, # Default lifecycle policy
-            "file_2": DELETED,
+            "file_2": DEEP_ARCHIVE,
```

Comment:
Why did this change?

Reply:
I removed the `EXPIRE_AFTER` constants in the appropriate default, long-term lifecycle policies, so the file should no longer be deleted after this length of time and should remain in DA.

We decided not to impose a strict deletion time-frame in our default, long-term lifecycle policies, as long-term kind of implies the files will persist and we did not want to be caught off-guard in a few years with many files deleted under our nose.